ansible/roles-infra/infra-images/defaults/main.yaml

```diff
@@ -8,6 +8,13 @@ infra_images_redhat_owner_id: 309956199498
 
 infra_images_predefined:
 
+  RHEL91GOLD-latest:
+    owner: "{{ infra_images_redhat_owner_id }}"
+    name: RHEL-9.1*_HVM-*Access*
```

Comment:
if one day there is 9.10, that will catch it.

maybe `    name: RHEL-9.1.*_HVM-*Access*`